### DISCUSSION TITLE
2.0.2 version

### DISCUSSION THREAD

**Discussion by tonylu110** [tonylu110/uyou-todo-electron#12] [Announcements]
- new ToDo item UI
- Added run in the background
- Added hide ToDo item button
- Added set ToDo Reminder

<hr /><em>This discussion was created from the release <a href='https://github.com/tonylu110/uyou-todo-electron/releases/tag/2.0.2'>2.0.2 version</a>.</em>

**Comment by liufang185** [comment]
WIN7有问题。报错2.0.3和2.0.4 都不行啊 ，，报错：“无法定位程序输入点DiscardVirtualMemory于动态链接库KERNEL32.dll，求解决
